server/pictureGraber.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
```

Comment:
pictureGrabber.js perhaps? Two 'b's?
Would need to be changed wherever it's being called if you update it.
Minor, non-blocking.

Reply:
I believe you are correct.